frontend/test/metabase/scenarios/onboarding/search/recently-viewed.cy.spec.js

```diff
@@ -15,6 +15,7 @@ describe(`search > recently viewed`, () => {
 
     // "Orders in a dashboard" dashboard
     cy.visit("/dashboard/1");
+    cy.findByText("Product ID");
```

Comment:
Not sure why this test started failing with this PR, but adding a page-load wait make it passes.

Failed run: https://app.circleci.com/pipelines/github/metabase/metabase/22566/workflows/2ed9b4b4-4630-4f4f-942b-8d07f6c85618/jobs/1000906

Reply:
FYI @nemanjaglumac